tests/conftest.py

```diff
@@ -31,18 +31,22 @@ def __init__(self, pixel_array, slice_position, row_cosine=None, column_cosine=N
 
         shape = pixel_array.shape
         if len(shape) == 2:
-            num_columns, num_rows = shape
+            num_rows, num_columns = shape
             samples_per_pixel = 1
         else:
-            num_columns, num_rows, samples_per_pixel = shape
+            num_rows, num_columns, samples_per_pixel = shape
+            # TODO: add
```

Comment:
If you don't mind, could you add a little more context for this TODO?

Reply:
Sure, I added information in the comment.
This TODO is just about supporting a rare (nonexistent maybe?) but allowed situation of planar configuration = 1, which means that the RGB channels are along the first axis of `pixel_array`.
Usually, planar configuration = 0, and the channels are last.
Anyway, a dicom series with RGB slices is itself a pretty uncommon situation, though dicom-numpy supports it.